docs/sources/auth/generic-oauth.md

```diff
@@ -235,5 +240,38 @@ Payload:
 
 Config:
 ```bash
-role_attribute_path = contains(info.groups[*], 'admin') && 'Admin' || contains(info.groups[*], 'editor') && 'Editor' || 'Viewer'
+role_attribute_path = contains(info.roles[*], 'admin') && 'Admin' || contains(info.roles[*], 'editor') && 'Editor' || 'Viewer'
+```
+
+
+### Groups mapping
+
+>  Available in Grafana Enterprise v8.1 and later versions.
+
+With Team Sync you can map your Generic OAuth groups to teams in Grafana so that the users are automatically added to the correct teams.
+
+Generic OAuth groups can be referenced by group ID, like `8bab1c86-8fba-33e5-2089-1d1c80ec267d` or `myteam`.
+
+[Learn more about Team Sync]({{< relref "team-sync.md" >}})
+
+Config:
+
+```bash
+groups_attribute_path = info.groups.join(',',[*])  
```

Comment:
Here's where it would make more sense to me if this was just `groups_attribute_path = info.groups`, i.e. a string array